actions/lib/action.py

```diff
@@ -60,6 +59,12 @@ def _get_st2_urls(self):
 
     def _get_api_key(self):
         api_key = self.config.get('api_key', None)
+
+        # not found look up from env vars. Assuming the pack is
+        # configuered to work with current StackStorm instance.
+        if not api_key:
+            api_key = os.environ.get('ST2_ACTION_AUTH_API_KEY', None)
```

Comment:
To be consistent with the rest of StackStorm this should be `ST2_API_KEY` i believe

Reply:
I took the name from the previously existing ones. So I think all of them should be renamed then?
```python
        if not base_url:
            api_url = os.environ.get('ST2_ACTION_API_URL', None)
            auth_url = os.environ.get('ST2_ACTION_AUTH_URL', None)
```
and
```python
        if not token:
            token = os.environ.get('ST2_ACTION_AUTH_TOKEN', None)
```

Reply:
Ah sorry, i didn't see the other variables in here. This is fine